app/styles/tables.less

```diff
@@ -0,0 +1,83 @@
+.table-detail {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.table-detail-row {
```

Comment:
please follow the BEM naming convention block__item--modifier
http://getbem.com/naming/
example: .table-detail__row,  .table-detail__cell ,  .table-detail__cell__title etc